assets/src/js/cookies.js

```diff
@@ -156,12 +156,21 @@ export const setupCookies = () => {
 
       // Update ad storage and analytics storage if Google Consent Mode is enabled
       if (ENABLE_GOOGLE_CONSENT_MODE) {
-        updateGoogleConsent({
-          ad_storage: marketingCookiesChecked ? 'granted' : 'denied',
-          ad_user_data: marketingCookiesChecked ? 'granted' : 'denied',
-          ad_personalization: marketingCookiesChecked ? 'granted' : 'denied',
-          ...ENABLE_ANALYTICAL_COOKIES && {analytics_storage: analyticalCookiesChecked ? 'granted' : 'denied'},
-        });
+        if (ENABLE_ANALYTICAL_COOKIES) {
```

Comment:
When analytical cookies functionality is disabled, we simply don't update `analytics_storage` and it will stay on the default value that was set before the cookie box was clicked. Before the change, it was always updated to `denied` if the functionality is disabled.